src/Components/Homepage/MapSection.jsx

```diff
@@ -13,7 +13,7 @@ import Card from "@mui/material/Card";
 import Button from "@mui/material/Button";
 import Menu from "@mui/material/Menu";
 import { NavigationFormValidator } from "./NavigationFormValidator.jsx";
-import { MapContainer, TileLayer, useMap, Marker, Popup } from "react-leaflet";
+// import { MapContainer, TileLayer, useMap, Marker, Popup } from "react-leaflet";
```

Comment:
Are you sure these are not needed? please consult with @Jeanndo before commenting the lines out